Ref/src/adapter/eps/hardware/batteryLevel/component/epsBatteryComponent.fpp

```diff
@@ -0,0 +1,80 @@
+module Ref {
+
+  @ EPS Battery Logic Component (in adapter layer)
+  active component epsBatteryComponent {
+
+    # ----------------------------------------------------------------------
+    # General ports
+    # ----------------------------------------------------------------------
+
+    @ Port for sending the battery level to the controller component
+    output port epsBatLogicOut: eps_logic_bat_output
+
+    @ Port for receiving the result
+    sync input port epsBatLogicIn: eps_logic_bat_input
+
+    # ----------------------------------------------------------------------
+    # Special ports
+    # ----------------------------------------------------------------------
+
+    @ Command receive port
+    command recv port cmdIn
+
+    @ Command registration port
+    command reg port cmdRegOut
+
+    @ Command response port
+    command resp port cmdResponseOut
+
+    @ Event port
+    event port eventOut
+
+    @ Telemetry port
+    telemetry port tlmOut
+
+    @ Text event port
+    text event port textEventOut
+
+    @ Time get port
+    time get port timeGetOut
+
+    # ----------------------------------------------------------------------
+    # Commands
+    # ----------------------------------------------------------------------
+
+    @ Send battery level
+    async command SEND_BATTERY_LEVEL(
+                           battery: F32 @< the battery level to send through
+                         )
```

Comment:
Also should go

Reply:
old battery component, can ignore

Reply:
if these are old the whole component should be removed from the codebase - we should not have dead code living outside of the commit log

Reply:
if we ever need to view the old code we can just look on github @NishqR

Reply:
ye good point, agreed